src/trans-netlist/trans_to_netlist.cpp

```diff
@@ -593,6 +593,9 @@ void convert_trans_to_netlistt::convert_lhs_rec(
   // default
   forall_operands(it, expr)
   {
+    if(it->type().id() == ID_natural || it->type().id() == ID_integer)
+      continue;
```

Comment:
Curious/would appreciate a comment in the code: Why is "ignore this" the right way to go here?

Reply:
Done -- this whole function should be replaced by a proper dependency graph, which could be reused elsewhere.